reconcile/prometheus_rules_tester/integration.py

```diff
@@ -136,6 +137,15 @@ def get_rules_and_tests(
         prom_rules = [
             r for r in namespace["openshiftResources"] if r["provider"] in PROVIDERS
         ]
+        prom_rules += [
+            r
+            for r in list(
+                itertools.chain.from_iterable([
+                    s["openshiftResources"] for s in namespace["sharedResources"]
+                ])
+            )
+            if r["provider"] in PROVIDERS
+        ]
```

Comment:
I don't understand why this is needed... `orb.get_namespaces` should take care of shared resources, right?

Reply:
same question, it should be done in https://github.com/app-sre/qontract-reconcile/blob/d26fb63665718895913b6e4ad5ce0067155ece95/reconcile/openshift_resources_base.py#L805 already, or any edge cases missing for this?

Reply:
You are right. perhaps I was to deep into it and got confused :see_no_evil: . Let me check again

Reply:
Yes, it's not needed